x-pack/plugins/observability_solution/slo/public/context/plugin_context.tsx

```diff
@@ -16,14 +16,8 @@ export interface PluginContextValue {
   isServerless?: boolean;
   appMountParameters?: AppMountParameters;
   observabilityRuleTypeRegistry: ObservabilityRuleTypeRegistry;
-  ObservabilityPageTemplate: React.ComponentType<LazyObservabilityPageTemplateProps>;
+  ObservabilityPageTemplate?: React.ComponentType<LazyObservabilityPageTemplateProps>;
```

Comment:
@shahzad31 I don't know how to proceed with the ts error I got regarding `ObservabilityPageTemplate`. Any ideas?